UC4-chaincode/src/test/java/de/upb/cs/uc4/chaincode/ExamContractTest.java

```diff
@@ -93,4 +96,18 @@ private Executable getExamsSuccessTest(
         };
     }
 
+    private Executable getExamsFailureTest(
+            JsonIOTestSetup setup,
+            List<String> input,
+            List<String> compare,
+            List<String> ids
+    ) {
+        return () -> {
+            Context ctx = TestUtil.buildContext(GroupContract.contractName, GroupContract.transactionNameGetUsersForGroup, setup, input, ids);
+
+            String result = contract.getExams(ctx, input.get(0), input.get(1), input.get(2), input.get(3), input.get(4), input.get(5), input.get(6));
+            assertThat(result).isEqualTo(compare.get(0));
+        };
+    }
+
```

Comment:
```suggestion
```
This is not needed, as the test structure for successful and unsuccessful get requests is the same.